chart/compass/values.yaml

```diff
@@ -43,6 +43,9 @@ global:
 
   director:
     port: 3000
+    assignedScopes:
```

Comment:
Please remove it and define it in config.yaml (old scopes.yaml)

Reply:
Waiting for the #365 to be resolved then.